.github/workflows/build.yml

```diff
@@ -0,0 +1,40 @@
+name: build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test-and-build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up go
+        uses: actions/setup-go@v2-beta
+        with:
+          go-version: '1.13'
+      - name: Set up go env
+        run: |
+          echo "::set-env name=GOPATH::$(go env GOPATH)"
+          echo "::add-path::$(go env GOPATH)/bin"
+        shell: bash
+      - name: Install jq
+        run: |
+          mkdir -p deps/bin
+          curl -s -L -o deps/bin/jq https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64
+          chmod +x deps/bin/jq
+          echo "::add-path::${PWD}/deps/bin"
+      - name: Test
+        run: make test
+      - name: Build
+        run: |
+          make build-linux
+          make package-linux
+      - uses: actions/upload-artifact@v1
+        with:
+          name: lifecycle
+          path: out/lifecycle-v0.0.0+linux.x86-64.tgz
```

Comment:
Can we change from hardcoding `0.0.0`, to hardcoding the next version `0.7.0`, in the action and in the Makefile? I think that would be better until we set up automation to bump the version after releases.

Reply:
Does `path` support globs here (i.e. `out/lifecycle-v*+linux.x86-64.tgz`)?

Reply:
> Does path support globs here (i.e. out/lifecycle-v*+linux.x86-64.tgz)?

No, however https://github.com/actions/upload-artifact/issues/11 has recent-ish activity, so perhaps soon?

We could use variables rather than hardcoding, which would be a step towards automation of the bumpings